<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sinema</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0,0,0)</string>
  </property>
  <property name="iconSize">
   <size>
    <width>40</width>
    <height>40</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="main_foto_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/newPrefix/images/sinema.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/newPrefix/images/sinema.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="film_ekle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 249, 60)</string>
    </property>
    <property name="text">
     <string>Film Ekle</string>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/newPrefix/images/ekle3.png</normaloff>:/newPrefix/images/ekle3.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="seans_ekle">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 249, 60)</string>
    </property>
    <property name="text">
     <string>Seans Ekle</string>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/newPrefix/images/ekle3.png</normaloff>:/newPrefix/images/ekle3.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="salon_ekle">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 249, 60)</string>
    </property>
    <property name="text">
     <string>Salon Ekle</string>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/newPrefix/images/ekle3.png</normaloff>:/newPrefix/images/ekle3.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
